### PR TITLE
Partial fix non-inverted menu images at inet.se

### PR DESCRIPTION
**Pull request by AnonymusElephant**
Fixed non-inverted menu images in sidemenu and submenues. Could not track down why some images unloads (problem existed before this fix).

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -9722,6 +9722,19 @@ IGNORE INLINE STYLE
 
 ================================
 
+inet.se
+
+INVERT
+.h1yh8nho
+.i1gv55ha
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9729]: Why does this element needs to be inverted?

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -9722,6 +9722,19 @@ IGNORE INLINE STYLE
 
 ================================
 
+inet.se
+
+INVERT
+.h1yh8nho
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9728]: How can I find this element? This seems to be hidden by default.

File: src/config/dynamic-theme-fixes.config [hunk 3]
```diff
@@ -9722,6 +9722,19 @@ IGNORE INLINE STYLE
...
+.i1gv55ha
+
+CSS
+:root {
+--svg-icon-default-color: #e8e6e3
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:9733]:
```suggestion
    --svg-icon-default-color: #e8e6e3 !important;
```

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (AnonymusElephant) [reply src/config/dynamic-theme-fixes.config:9729]: It's the icons for the different categories in menus on the site, that are black if not inverted which makes them difficult to see.

File: src/config/dynamic-theme-fixes.config (hunk 2 above)
Comment (AnonymusElephant) [reply src/config/dynamic-theme-fixes.config:9728]: It's the compact menu selector that only appears when site is in a narrow window to back or show/hide main categories. Same issue here as with icons tha it's black by default and hard to see

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:9729]:
```suggestion
a.item-button span[class*="icon"]
```

File: src/config/dynamic-theme-fixes.config (hunk 3 above)
Comment (AnonymusElephant) [reply src/config/dynamic-theme-fixes.config:9733]: Oh yes! I forgot the "important!" line, thanks!

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (AnonymusElephant) [reply src/config/dynamic-theme-fixes.config:9729]: That works better, thanks! Still find that when turning on darkmode in dark reader some icon in main side menu first loads in correctly to then unload and be replaced by none or broken icon elements (was a problem before making corrections to dark theme)

**Comment by AnonymusElephant** [comment]
Fixed push with sugestions form Gusted

**Review by Gusted — APPROVED** [review]

**Pull request merged**